hash/apple2gs_flop_orig.xml

```diff
@@ -4086,4 +4079,150 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="sinbadfal">
+		<description>Sinbad and the Throne of the Falcon</description>
+		<year>1990</year>
+		<publisher>Cinemaware</publisher>
+		<info name="usage" value="Requires a 768K Apple IIgs ROM1 or later."/>
+		<sharedfeat name="compatibility" value="A2GS" />
+		<!-- Dump released: 2022-11-20 -->
+		<!--"Sinbad and the Throne of the Falcon" is a 1990 action game developed by Curt Toumanian (art director), Steve Quinn (original art), Jeff Hilbers (art), Rob Landeros (art), Russ Truelove (art), Bill Williams (music & story), Jim Simmons (orchestration), Andrew Caldwell (orchestration), Douglass Sharp (computagraphy), and distributed by Cinemaware.-->
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 1 Reel 1" />
+			<dataarea name="flop" size="1296414">
+				<rom name="sinbad and the throne of the falcon - reel 1.woz" size="1296414" crc="a3056453" sha1="f90a3f48445f80abe28ba368ff9de51deb55930a" />
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 2 Reel 2 " />
+			<dataarea name="flop" size="1296381">
+				<rom name="sinbad and the throne of the falcon - reel 2.woz" size="1296381" crc="3eb2b298" sha1="1920ab0321f8bcf85a607baf2b3a0778251c3747" />
+			</dataarea>
+		</part>
```

Comment:
(Sinbad and the Throne of the Falcon, line 4090) The part ID for the second disk has trailing space.